.github/workflows/ruby.yml

```diff
@@ -0,0 +1,67 @@
+name: Ruby
+
+env:
+  TZ: Europe/London
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+
+    env:
+      RAILS_ENV: test
+
+    services:
+      mysql:
+        # Use the Mysql docker image https://hub.docker.com/_/mysql
+        image: mysql:5.7
+        ports:
+         - 3306 # Default port mappings
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+        env:
+            MYSQL_ALLOW_EMPTY_PASSWORD: yes
+            MYSQL_ROOT_PASSWORD: ''
+            MYSQL_DATABASE: labwhere_test
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        bundler-cache: true # Runs bundle install and caches gems. See the ruby_test.yml
+                            # example if you need more control over bundler.
+    - name: Set up environment
+      env:
+        DBPORT: ${{ job.services.mysql.ports[3306] }}
+      run: |
+        cp config/database.yml.example config/database.yml
+        cp config/secrets.yml.example config/secrets.yml
+        cp config/bunny.yml.example config/bunny.yml
+        bundle exec rails db:setup
+        bundle exec rails about
+
```

Comment:
`bundle exec rails about`
Ooh, that's a nice addition. 
I wonder if its worth adding `bundle exec rails stats` as well.
(I was going to suggest bringing it into a separate step to make it easier to find, but looks like db:setup for labwhere is not as chatty as it is for Sequencescape)